src/views/MondoMarket/index.jsx

```
@@ -0,0 +1,228 @@
+  const [mondoCategories, setMondoCategories] = useState([]);
+  const [items, setItems] = useState([]);
+
+  const [topMondo, setTopMondo] = useState(null);
+  const [activeRegion, setActiveRegion] = useState(null);
+  const [currentMondo, setCurrentMondo] = useState(null);
+
+  // ** Redux ---
+  const dispatch = useDispatch();
+  const { bgColor } = useSelector((state) => state.theme);
+  const { token } = useSelector((state) => state.auth);
+  const { isFetching, isTopFetching, isMondoFetching, isSearching } =
```

ESLint

[error] 'isSearching' is assigned a value but never used.

(no-unused-vars)